### DISCUSSION TITLE
Welcome to ultimate-backend Discussions!

### DISCUSSION THREAD

**Discussion by juicycleff** [juicycleff/ultimate-backend#160] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bpofficial** [comment]
Hello I have a question regarding the login procedure for the server :)
When you log in via the `Password` service type, what sort of user-stored authorization is supposed to be used? Does it create an access token for the logged in user to use (ie. JWT)? Or does it create a cookie? I can't seem to find the logic behind what actually happens upon login, rather than just return the user and machine details.

Ps. I have noticed that the login returns a unique ID, is this a session ID? If so, where is it applicable? For example, I have created my account, verified my details, logged in etc. Now, I'd like to add some extra infomation to my account, i.e. add a phone number & verify it. How would I verify my identity to the server post `Password` login?


Cheers !

**Reply by bpofficial** [reply]
I've figured it out, after getting everything to work and playing around with the GQL endpoints i saw a session get created as a cookie. I'll hopefully have some time to document the things i've learnt with this repo and create a doc pull request :D

Thanks for your work on this project, It's a steep learning curve initially but I think i've gotten the hang of the structure.

**Comment by irl-dan** [comment]
Hi! Thanks for building this, this is cool. Is this considered stable and/or is a fork of it being used in production by any other members of the community?

**Comment by Oceans22DAO** [comment]
Looking forward to learning from this backend!
